website_google_tag_manager/views/res_config_settings_view.xml

```diff
@@ -8,48 +8,40 @@
         <field name="model">res.config.settings</field>
         <field name="inherit_id" ref="website.res_config_settings_view_form" />
         <field name="arch" type="xml">
-            <div id="plausbile_setting" position="after">
-                <div
-                    class="col-xs-12 col-lg-6 o_setting_box"
-                    id="google_tag_manager_setting"
-                >
-                    <div class="o_setting_right_pane">
-                        <label
-                            for="google_tag_manager_key"
-                            string="Google Tag Manager"
-                        />
-                        <span
-                            class="fa fa-lg fa-globe"
-                            title="Values set here are website-specific."
-                            groups="website.group_multi_website"
-                        />
-                        <div class="text-muted">
-                           Manage all your tags without editing code.
-                        </div>
-                        <div class="content-group">
-                            <div class="row mt16">
-                                <label
-                                    class="col-lg-4 o_light_label"
-                                    for="google_tag_manager_key"
-                                />
-                                <field
-                                    name="google_tag_manager_key"
-                                    placeholder="GTM-XXXXX"
-                                />
-                            </div>
-                        </div>
-                        <div>
-                            <a
-                                href="https://support.google.com/tagmanager/answer/6103696#install?hl=en"
-                                class="oe_link fa fa-arrow-right"
-                                target="_blank"
-                            >
-                                How to get my GTM container ID
-                            </a>
+            <setting id="plausbile_setting" position="after">
+                <setting id="google_tag_manager_setting">
+                    <label for="google_tag_manager_key" string="Google Tag Manager" />
+                    <span
+                        class="fa fa-lg fa-globe"
+                        title="Values set here are website-specific."
+                        groups="website.group_multi_website"
+                    />
```

Comment:
Odoo has removed these hints since 16.0, as now there's a header indicating that the settings are for the corresponding website, so better to remove it.

```suggestion
```

Reply:
Done the suggested change. Please review.